src/common/constants/pool-constants.ts

```diff
@@ -0,0 +1,17 @@
+export const DeprecatedContainerImages = [
```

Comment:
Can this go as a utility function in `util/pool-utils.ts`?

Something like:

isDeprecatedContainerImage: (imageId: string): boolean {
    // Check imageId.toLowerCase() is in the list of deprecated images
}

Reply:
Yeah that's a good call, I wasn't sure where to put that list of deprecated container images at first which is why I made a new constants file. I changed it to your suggestion now.